boa/src/builtins/property/mod.rs

```diff
@@ -333,26 +351,35 @@ impl fmt::Display for PropertyKey {
         match self {
             PropertyKey::String(ref string) => string.fmt(f),
             PropertyKey::Symbol(ref symbol) => symbol.fmt(f),
+            PropertyKey::Index(index) => index.fmt(f),
         }
     }
 }
 
-impl From<&PropertyKey> for RcString {
-    #[inline]
-    fn from(property_key: &PropertyKey) -> RcString {
-        match property_key {
-            PropertyKey::String(ref string) => string.clone(),
-            PropertyKey::Symbol(ref symbol) => symbol.to_string().into(),
-        }
-    }
-}
+// impl From<&PropertyKey> for RcString {
+//     #[inline]
+//     fn from(property_key: &PropertyKey) -> RcString {
+//         match property_key {
+//             PropertyKey::String(ref string) => string.clone(),
+//             PropertyKey::Symbol(ref symbol) => symbol.to_string().into(),
+//             PropertyKey::
+//         }
+//     }
+// }
```

Comment:
Do we need this?

Reply:
Not really, this caused a very hard bug .